ra-tls/pal/Makefile

```diff
@@ -61,7 +62,7 @@ endif
 
 #App_C_Files := App.c server-tls.c sgxsdk-ra-attester_u.c ias-ra.c
 #App_C_Files := App.c sgxsdk-ra-attester_u.c
-App_C_Fils := App.c
+App_C_Files := App.c
```

Comment:
Why rename to App_C_Files?

Reply:
it should be typo. App_C_Files not App_C_Fils